modules/monitoring-configuring-remote-write.adoc

```diff
@@ -0,0 +1,176 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="configuring_remote_write_storage_{context}"]
+= Configuring remote write storage
+
+You can configure remote write to enable Prometheus to send ingested metrics to remote systems for storage. 
+Doing so has no impact on how or for how long Prometheus stores metrics using local storage.
```

Comment:
'local storage' as a term has special implications. Consider using the term `defined storage` or `configured storage medium`.

Reply:
Good point. Will update the wording to clarify,

Reply:
per QE, changing to `persistent storage`